test/test.sh

```
@@ -66,7 +66,7 @@ for chart in "$@"; do
     echo charts/$chart tests PASSED
     echo
     echo results:
-    $kc logs test-$chart
```

this will ignore the error right? why? will we catch it later?

---

We don't have a resource named `test-agent` in the agent chart, so there's nothing to get logs from. I don't think this was ever testing anything on purpose because it just calls `kubectl logs`.